src/app.html

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="light">
```

Comment:
ik heb soms dat bij ctrl+f5 dat hij terug valt op lightmode, miss kunnen we dit opslaan in localStorage?